contrib/containerized-pbench/setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [[ -z "${1}" ]]; then
+    echo "Please provide a target Pbench Server (<host name>:<port number>) to move the results to as the first argument." >&2
+    exit 1
+fi
+cat > ./pbench-agent.cfg.tmp << EOF
+[DEFAULT]
+pbench_install_dir = /opt/pbench-agent
+pbench_web_server = ${1}
+[config]
+path = %(pbench_install_dir)s/config
+files = pbench-agent-default.cfg
+EOF
```

Comment:
Consider using `<<-` instead of `<<` and then prefixing the lines up to and including the `EOF` line with tab characters.

Reply:
Fixed.